var/spack/repos/builtin/packages/lorene/package.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Lorene(MakefilePackage):
+    """LORENE: Langage Objet pour la RElativite NumeriquE
+
+    LORENE is a set of C++ classes to solve various problems
+    arising in numerical relativity, and more generally in
+    computational astrophysics. It provides tools to solve
+    partial differential equations by means of multi-domain
+    spectral methods."""
+
+    homepage = "https://lorene.obspm.fr/index.html"
+    cvs      = ":pserver:anonymous:anonymous@octane.obspm.fr:/cvsroot%module=Lorene"
+
+    maintainers = ['eschnett']
+
+    version('2021.4.22', date='2021-04-22')
+
+    variant('fftw', default=True)
+    variant('Bin_star', default=True)
+    # variant('Bin_star_ncp', default=True)
+
+    patch('local_settings')
```

Comment:
Can you add a comment above this directives to explain what it does and make the provenance explicit?

Reply:
Done.